README.md

```diff
@@ -95,6 +95,9 @@ including Citadel (acting as Certificate Authority), citadel agent, etc.
 extensions to the [Envoy proxy](https://github.com/envoyproxy/envoy) (in the form of
 Envoy filters) that support authentication, authorization, and telemetry collection.
 
+- [istio/ztunnel](https://github.com/istio/ztunnel). The repository contains the Rust implementation of the Ztunnel
```

Comment:
Looking, it seems there is no capitalization within ztunnel, so I suspect this should be:
```suggestion
- [istio/ztunnel](https://github.com/istio/ztunnel). The repository contains the Rust implementation of the ztunnel
```